### ISSUE TITLE
Browser tab opens several seconds slower when alarm sound is enabled

### ISSUE THREAD

**Issue by NathanR-85** [camtheperson/get-my-ps5#27]
Maybe it's waiting for the MP3 to finish playing?

**Comment by baileygranam** [comment]
I found that if you move the `open()` function call above
```
if (alarm.toUpperCase() === "Y") {
            playAlarm();
        }
```

it will open the browser immediately first, then play the sound

**Comment by damm** [comment]
Yes it took about 5-6 seconds before the window opened.

**Comment by outphase84** [comment]
Planned fix in 1.2 -- in the interim, you can fix this by modifying index.js and changing this:

```
  const onSuccess = () => {
        console.log("Found it! Opening queue now...");
        if (alarm.toUpperCase() === "Y") {
            playAlarm();
        }
        open(playstationType[choice].url);
    };
```

to this:

```
    const onSuccess = () => {
        console.log("Found it! Opening queue now...");
        open(playstationType[choice].url);
        if (alarm.toUpperCase() === "Y") {
            playAlarm();
        }        
    };
```

**Comment by NathanR-85** [comment]
Thanks, did this already based on @baileygranam's suggestion. But glad you confirmed it will work - time is of the essence!
